scripts/deploy/medic-deploy.md

```diff
@@ -0,0 +1,76 @@
+## Medic Deployment
+
+| Key       | Value                                                   |
+|-----------|---------------------------------------------------------|
+| context   | arn:aws:eks:eu-west-2:720541322708:cluster/prod-cht-eks |
+| namespace | users-chis-prod                                         |
+
+### Requirements:
+1. Applications:
+  - [Helm](https://helm.sh/)
+  - [Kubectl](https://kubernetes.io/docs/reference/kubectl/kubectl/)
+
+2. [Add medic helm chart repository](https://github.com/medic/helm-charts/tree/main#usage)
+
+3. [Authenticate to Medic kubernetes cluster (EKS)](https://github.com/medic/medic-infrastructure/blob/master/terraform/aws/dev/eks/access/README.md)
+
+
+### Deploy new version
+
+#### KE
+```shell
+# Edit tag in users-chis-ke.yaml and then run:
+
+helm upgrade \
+      --kube-context arn:aws:eks:eu-west-2:720541322708:cluster/prod-cht-eks \
+      --namespace users-chis-prod \
+      --values values/users-chis-ke.yaml \
+      users-chis-ke medic/cht-user-management
```

Comment:
helm was yelling at me that `Error: repo medic not found` when I ran this. I had to specify a full path to the local repo like this:

```
helm upgrade       --kube-context arn:aws:eks:eu-west-2:7205
41322708:cluster/prod-cht-eks       --namespace users-chis-prod       --values values/users-chis-
ke.yaml       users-chis-ke /home/mrjones/Documents/MedicMobile/helm-charts/charts/cht-user-manag
ement/
```

should we change them all to be an explicit path?

Reply:
resolved in commit above about path to this chart repo.